web/src/lib/core/Stitch.ts

```diff
@@ -5,96 +5,112 @@ import StitchAppClient from "./StitchAppClient";
 
 const DEFAULT_BASE_URL = "https://stitch.mongodb.com";
 const TAG = "Stitch";
-const appClients: {[key: string]: StitchAppClientImpl} = {};
+const appClients: { [key: string]: StitchAppClientImpl } = {};
 
 export default class Stitch {
-    private static ensureInitialized() {
-      if (!Stitch.initialized) {
-        throw new Error("Stitch not initialized yet; please call initialize() first");
-      }
+  private static ensureInitialized() {
+    if (!Stitch.initialized) {
+      throw new Error(
+        "Stitch not initialized yet; please call initialize() first"
+      );
     }
+  }
 
-    public static initialize() {
-      if (Stitch.initialized) {
-        return;
-      }
-  
-      // TODO: get app info
-  
-      Stitch.initialized = true;
+  public static initialize() {
+    if (Stitch.initialized) {
+      return;
     }
-  
-    public static get defaultAppClient(): StitchAppClient {
-      Stitch.ensureInitialized();
-      if (Stitch.defaultClientAppId === undefined) {
-        throw new Error("default app client has not yet been initialized/set");
-      }
-      return appClients[Stitch.defaultClientAppId];
+
+    // TODO: get app info
```

Comment:
[nit] is there a ticket for this? if so can you add the number here so we don't forget about it?